drivers/interrupt_controller/intc_loapic.c

```diff
@@ -69,6 +69,25 @@ static uint32_t loapic_device_power_state = DEVICE_PM_ACTIVE_STATE;
 mm_reg_t z_loapic_regs;
 #endif
 
+#ifdef CONFIG_INTEL_VTD_ICTL
+
+#include <drivers/interrupt_controller/intel_vtd.h>
+static const struct device *vtd;
+
+static bool get_vtd(void)
+{
+	if (vtd != NULL) {
+		return true;
+	}
+#define DT_DRV_COMPAT intel_vt_d
```

Comment:
Don't we need to take a backup `DRV_COMPAT_BAK` here, like what was done on `intc_ioapic.c`

Reply:
Since there is any DRV_COMPAT being set for loapic it's unnecessary

Reply:
Ok, just confirming.